recipes-bsp/grub/grub/grub-runmode-bootimage.cfg

```diff
@@ -5,16 +5,10 @@ set kernel_path='/runmode/bzImage'
 
 set otherbootargs="rootwait rw usbcore.usbfs_memory_mb=0 consoleblank=0 rcu_nocbs=all "
 
-# These files are used to disable spectre/meltdown mitigations for
+# This file is used to disable CPU vulnerability mitigations for
 # performance reasons. nirtcfg refers to these paths specifically for
 # enabling/disabling these configuration fragments, so they cannot go
-# into /runmode/bootimage.d. We also cannot load them via wildcard
-# (e.g. cve-*.cfg) because the provisioned grub might not have the
-# regexp module that adds support for file globbing.
-for mitigation_cfg in cve-2017-5715 cve-2017-5754 cve-2018-3620_3646 cve-2018-3639; do
-	if [ -f "/runmode/${mitigation_cfg}.cfg" ]; then
-		source "/runmode/${mitigation_cfg}.cfg"
-	fi
-done
+# into /runmode/bootimage.d.
+source "/runmode/cpu-mitigations.cfg"
```

Comment:
if we're still doing the "rename to `.cfg.disable`" thing then this `source` should be guarded with an `if [ -f ... ]`